rest-dashboard.go

```diff
@@ -30,6 +30,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+const DefaultFolderId = 0
```

Comment:
This deserves a comment above it to tell our users what it means:
```
// DefaultFolderId is the id of the general folder that is pre-created and cannot be removed.
```
or something along those lines.